core-zipline/src/jsMain/kotlin/io/github/droidkaigi/confsched2022/zipline/JsScheduleModifier.kt

```diff
@@ -18,7 +18,8 @@ class JsScheduleModifier() : ScheduleModifier {
             dayToTimetable = schedule.dayToTimetable.mapValues { timetable ->
                 val modifiedSessions = timetable.value.timetableItems.map { timetableItem ->
                     if (timetableItem is Session &&
-                        timetableItem.id == TimetableItemId("1")
+                        // Day 1 lunch
+                        timetableItem.id == TimetableItemId("b8528bb4-284c-424e-8be5-a4c1721e4ba8")
```

Comment:
How about changing TBW to "To be written."? Because we can use it. 😄 

if(
...
timetableItem.id == TimetableItemId("b8528bb4-284c-424e-8be5-a4c1721e4ba8") &&
timetableItem.targetAudience == "TBW"
) {
...
 targetAudience = "To be written."

Reply:
Looks good!
Fixed at [74061d6](https://github.com/DroidKaigi/conference-app-2022/pull/456/commits/74061d6405ad6604968c21b442af7b6da2486479)